content/en/docs/topics/plugins.md

```diff
@@ -120,35 +121,105 @@ Restrictions on `name`:
 - `name` cannot duplicate one of the existing `helm` top-level commands.
 - `name` must be restricted to the characters ASCII a-z, A-Z, 0-9, `_` and `-`.
 
-`version` is the SemVer 2 version of the plugin. `usage` and `description` are
+### The `version` Field
+
+The `version` is the SemVer 2 version of the plugin. `usage` and `description` are
 both used to generate the help text of a command.
 
+### The `ignoreFlags` Field
+
 The `ignoreFlags` switch tells Helm to _not_ pass flags to the plugin. So if a
 plugin is called with `helm myplugin --foo` and `ignoreFlags: true`, then
 `--foo` is silently discarded.
 
-Finally, and most importantly, `platformCommand` or `command` is the command
-that this plugin will execute when it is called. The `platformCommand` section
-defines the OS/Architecture specific variations of a command. The following
-rules will apply in deciding which command to use:
+### The `platformCommand` Field
+
+The `platformCommand` configures the command that the plugin will execute when
+it is called. The following rules will apply in deciding which command to use:
 
 - If `platformCommand` is present, it will be searched first.
-- If both `os` and `arch` match the current platform, search will stop and the
+  - If both `os` and `arch` match the current platform, search will stop and the
   command will be used.
-- If `os` matches and there is no more specific `arch` match, the command will
-  be used.
-- If no `platformCommand` match is found, the default `command` will be used.
+  - If `os` matches and `arch` is empty, the command will be used.
+  - If `os` and `arch` are empty, the command will be used.
+- If no `platformCommand` match is found and the deprecated `command` is present
+  it will be used.
 - If no matches are found in `platformCommand` and no `command` is present, Helm
   will exit with an error.
 
+## Building a Plugin
+
+Here is the plugin YAML for a simple plugin that helps get the last release name:
+
+```yaml
+name: last
+version: 0.1.0
+usage: get the last release name
+description: get the last release name
+ignoreFlags: false
+platformCommand:
+  - command: ${HELM_BIN}
+    args:
+      - list
+      - --short
+      - --max=1
+      - --date
+      - -r
+```
+
+Plugins may require additional scripts, which can be included in the plugin directory,
+or executables, which require downloading via a hook. An example of the directory
+structure and plugin YAML for plugin based on an executable is below:
+
+```console
+$HELM_PLUGINS/
+  |- myplugin/
+    |- scripts/
+      |- install.ps1
+      |- install.sh
+    |- plugin.yaml
+```
+
+```yaml
+name: myplugin
+version: 0.1.0
+usage: example plugin
+description: example plugin
+ignoreFlags: false
+platformCommand:
+  - command: ${HELM_PLUGIN_DIR}/bin/myplugin
+  - os: windows
+    command: ${HELM_PLUGIN_DIR}\bin\myplugin.exe
```

Comment:
Does yaml handle these `\` or does it help to quote the strings here? I'm not a windows person.

Reply:
Strings in YAML rarely need quoting and `\` isn't an escape character in YAML strings.